src/main/java/com/rarchives/ripme/ripper/rippers/RedditRipper.java

```diff
@@ -27,7 +27,7 @@ public RedditRipper(URL url) throws IOException {
     private static final String HOST   = "reddit";
     private static final String DOMAIN = "reddit.com";
 
-    private static final String REDDIT_USER_AGENT = "RipMe:github/4pr0n/ripme:" + UpdateUtils.getThisJarVersion() + " (by /u/4_pr0n)";
+    private static final String REDDIT_USER_AGENT = "RipMe:github.com/RipMeApp/ripme:" + UpdateUtils.getThisJarVersion() + " (by /u/metaprime and u/ineedmorealts)";
```

Comment:
Cool. What is this actually used for? Why send a custom user agent to Reddit? And since this project was started by and owed largely to @4pr0n i.e. /u/4_pr0n I think we should keep his name on the list of authors.

Reply:
> Why send a custom user agent to Reddit?

Having a unique useragent is considered best practice when using the reddit api and the admins tend to rate limit non-unique useragents. Also putting our usernames in the useragent allows us to be contacted if there's something wrong with how ripme uses the reddit api, instead of the useragent just getting banned

Reply:
It should read `/u/ineedmorealts` instead of `u/ineedmorealts`.